backend/typescript/prisma/platformSignupsSeed.ts

```diff
@@ -0,0 +1,40 @@
+import { Prisma, PrismaClient, Status } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+async function main() {
+  const platformSignupPromises: Promise<Prisma.platformSignUpCreateInput>[] = [];
+
+  const users = await prisma.user.findMany();
+
+  // Check if there are users to associate with platform signups
+  if (users.length === 0) {
+    throw new Error("No users found for seeding platform signups");
+  }
+
+  const platformSignups: Prisma.platformSignUpCreateInput[] = [];
+  users.forEach((user, idx) => {
```

Comment:
for ESLint sake, can you please remove 'idx' since it's unused and type the User? You can just import { User} from "@prisma/client"

Reply:
Yessir 🫡